db/migrate/20220523173800_create_categories.rb

```diff
@@ -0,0 +1,11 @@
+class CreateCategories < ActiveRecord::Migration[7.0]
+  def change
+    create_table :categories do |t|
+      t.string :name
+      t.string :logo
+      t.references :user, null: false, foreign_key: true
+
```

Comment:
- [ ] Kindly make your foreign key have indexes. You can run another migration file just for indexes:

```
class AddIndexToTable < ActiveRecord::Migration
  def change
    add_index :table, :user_id
  end
end
```

Reply:
I don't think this si necessary, when I tried I got this error `ERROR:  relation "index_categories_on_user_id" already exists`